### ISSUE TITLE
Please Whitelist Google Drive

### ISSUE THREAD

**Issue by BrilliantBoo** [excalidraw/excalidraw#7768]
Google Drive links are not yet allowed to embed. Please whitelist google drive link to enable it to embed successfully. Thank you so much.

**Comment by dwelle** [comment]
I believe Google Suite domain isn't allowed to be embedded, but I can check again. What type of file are you trying to embed?

**Comment by BrilliantBoo** [comment]
Yes, I am unable to embed but when I tried, the suggestion that I should ask for the website to whitelisted on Github popped up. So I thought I'd do so, in case this an adjusted that can be done to the website. 
I'd like to embed an audio stored on Google Drive.
